File: LogicBuilding/PYTHON/forLoop.py

def print_number_1_to_10():
    for i in range(11):
        print(f"The number --> {i}")
  
def print_number_10_to_1():
    for i in range(10,0,-1):
        print(f"The number --> {i}")

def print_even_number(number):
    for i in range(number):
        if(i%2==0):
            print(f"The even number --> {i}")        
              
#print_number_10_to_1()
#print_number_1_to_10()
print_even_number(20)

File: LogicBuilding/JS/forLoop.js
//printNumber1to10();
//printNumber10to1();
printEvenNumber(14);
function printNumber1to10(){
    for(let i = 0 ;i<11;i++){
        console.log("The nubmer -->"+i)
    }
}

function printNumber10to1(){
    for(let i = 10 ;i>=0;i--){
        console.log("The nubmer -->"+i)
    }
}

function printEvenNumber(number){
    for(let i =0;i<=number;i++){
        if(i%2==0){
            console.log(`The even Number --> ${i}`)
        }

    }
}